engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -152,6 +159,21 @@
     private DisplayResolutionDependentFBOs displayResolutionDependentFBOs;
     private ShadowMapResolutionDependentFBOs shadowMapResolutionDependentFBOs;
 
+    private static RenderGraph renderGraph = new RenderGraph(); // TODO: Try making this non-static
+
+    // Required for the reflection magic that Systems use.
```

Comment:
Thanks @Cervator for the clarification. Can you please further clarify this paragraph:
>Also not sure if it is / should be a best practice that ES classes be structured in a way where they are never explicitly created from somewhere else (since Gestalt's angle is to auto-register stuff via reflection magic and annotations)

I'm not sure what the "never explicitly created" part would look like in practice?

That been said,:
1. I would still try to move the pre-existing constructor to postBegin(), so that there is only one constructor in place. 
2. I would also replace the `// Required for the reflection magic that Systems use.` comment with something more substantial, summarizing what Cervator wrote above in at least 2-3 lines.